pkg/runtime/scheme.go

```diff
@@ -421,6 +426,23 @@ func (s *Scheme) AddDefaultingFuncs(defaultingFuncs ...interface{}) error {
 	return nil
 }
 
+// AddTypeDefaultingFuncs registers a function that is passed a pointer to an
+// object and can default fields on the object. These functions will be invoked
+// when Default() is called. The newly added functions are invoked before older
```

Comment:
Awkward wording, if you invoke this twice for the same object the later call takes effect.